docs/admin/prerequisites/dns.rst

```diff
@@ -3,7 +3,7 @@
 DNS
 ====
 
-Renku needs a DNS.
+Renku requires a DNS domain.
```

Comment:
```suggestion
Renku requires a domain.
```

Reply:
I'd leave it there, I prefer to have a bit of redundancy to avoid ambiguity.

Reply:
ok